enhancements/network/allow-mtu-and-overlay-port-changes.md

```diff
@@ -0,0 +1,165 @@
+---
+title: allow-mtu-and-overlay-port-changes
+authors:
+  - "@juanluisvaladas"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-01-25
+last-updated: 2021-01-25
+status: provisional
+---
+
+# Allow MTU and overlay port changes
+
+This covers adding the capability to the cluster network operator of changing
+the MTU, and the vxlan and geneve ports post installation.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Customers may need to change the MTU, or the ports used for vxlan or geneve
+tunnels post-installation. However these changes aren't trivial and may cause
+downtime, hence the CNO forbids currently them.
+
+We propose a brand new daemonset that will run on every node of the cluster and
+will make the necessary changes in an ordered, and coordinated manner with a
+service disruption within the TCP timeout.
+
+## Motivation
+
+While customers usually set the MTU, and the overlay ports some times they
+need to change them afterwards for unpredictable reasons such as performance,
+or changes in the underlay.
+
+## Goals
+
+* Allow to change MTU post install on both OpenShift SDN and OVN Kubernetes.
+
+* Allow to change the overlay network ports on the underlay in both OpenShift
+  SDN and OVN Kubernetes.
+
+* Allow to change both vxlan and geneve ports in a single operation.
```

Comment:
"On OVN Kubernetes, allow changing both the hybrid overlay network VXLAN port and the pod network Geneve port in a single operation"

(ok so I didn't complain about this anywhere else, but if you fixed the capitalization of "VXLAN" and "Geneve" everywhere, I wouldn't hate it.)